client/package.json

```diff
@@ -8,32 +8,32 @@
     "build": "build"
   },
   "dependencies": {
-    "@babel/core": "^7.0.0",
-    "@babel/plugin-proposal-class-properties": "^7.10.4",
-    "@babel/plugin-proposal-decorators": "^7.10.5",
-    "@babel/plugin-proposal-object-rest-spread": "^7.3.1",
-    "@babel/plugin-syntax-dynamic-import": "^7.2.0",
-    "@babel/polyfill": "^7.2.5",
-    "@babel/preset-env": "^7.3.1",
-    "@babel/preset-react": "^7.0.0",
+    "@apollo/client": "^3.2.5",
+    "@babel/core": "^7.12.3",
+    "@babel/plugin-proposal-class-properties": "^7.12.1",
+    "@babel/plugin-proposal-decorators": "^7.12.1",
+    "@babel/plugin-proposal-object-rest-spread": "^7.12.1",
+    "@babel/plugin-syntax-dynamic-import": "^7.8.3",
+    "@babel/polyfill": "^7.12.1",
+    "@babel/preset-env": "^7.12.1",
+    "@babel/preset-react": "^7.12.5",
     "@nivo/bar": "^0.52.1",
     "@nivo/circle-packing": "^0.52.1",
     "@nivo/line": "^0.52.1",
     "@nivo/pie": "^0.52.1",
     "@nivo/stream": "^0.52.1",
-    "apollo-cache-inmemory": "^1.4.2",
-    "apollo-client": "^2.4.12",
-    "apollo-link-http": "^1.5.9",
     "axios": "^0.19.2",
-    "babel-loader": "^8.0.5",
+    "babel-loader": "^8.1.0",
     "bootstrap": "^3.4.1",
     "bundle-stats": "^1.10.1",
-    "circular-dependency-plugin": "^5.0.2",
+    "bundle-stats-webpack-plugin": "^2.3.0",
```

Comment:
You added `bundle-stats-webpack-plugin`, but it looks like the webpack config still imports from `bundle-stats`?

Bundle stats output is junked either way:
http://dev.ea-ad.ca/webpack5/bundle-stats.html

(might be that the stats snapshot it wants to compare to from master isn't compatible with webpack 5 stats output eitherway, but I still think it might be that the new plugin isn't actually being used)